backend/core/views.py

```diff
@@ -144,6 +149,20 @@ def summary_stats(self, request, pk=None):
     # def shuffle_reviewers(self, request, pk=None):
     #     return Response("TODO")
 
+    @action(
+        detail=False,
+        methods=["POST"],
+        serializer_class=CardSummarySerializer,
+        permission_classes=[HasObjectPermission(permissions=Team.PERMISSION_MANAGE_CARDS)]
+    )
+    def bulk_set_due_dates(self, request, pk=None):
+
+        team: models.Team = models.Team.objects.filter(name=request.data.get('team')).first()
+        team_cards = get_team_cards(team, request.data.get('content_item'))
```

Comment:
I've changed this a bit so that I get the content_item from the serializer.  Not so sure about the flavours part though cause all members of a team will all be Python devs or Javascript devs etc, do we then still need to check flavours?